### DISCUSSION TITLE
Sistema non supportato - Problemi con Systemd Journal

### DISCUSSION THREAD

**Discussion by Gioway** [indomus/forum#3072] [HUB: Home Assistant]
Ciao ,
chiedo aiuto per risolvere un problema che è comparso dalla versione 2022.11, non ho mai avuto problemi ma dalla versione 2024.5 non posso più visualizzare i log dei componenti aggiuntivi.
Ho già provato la procedura descritta nel link di supporto per risolvere ( https://www.home-assistant.io/more-info/unsupported/systemd_journal) ma senza successo, l'istallazione va a buonfine ma nulla cambia al riavvio dell'host.
Installazione SUPERVISED su debian 11.
Grazie.
![Screenshot_20240506-175421](https://github.com/indomus/forum/assets/70282399/0c6df42e-e2b9-4a63-b9d6-067f65d35479)
![Screenshot_20240506-175435](https://github.com/indomus/forum/assets/70282399/75392399-0e1b-4eaf-b0dc-c4ad1d0f41bc)

**Comment by snakuzzo** [comment]
_**If you still see this issue then run the [supervised installer](https://github.com/home-assistant/supervised-installer) on the host as the supervisor service may need an update as well.**_

**Reply by Gioway** [reply]
Fatto 3 volte, nada

**Reply by snakuzzo** [reply]
pazienza

**Reply by KC-inDomus** [reply]
passa a core...
